### DISCUSSION TITLE
What‘s the meaning of "Coolant" in the mail filter?

### DISCUSSION THREAD

**Discussion by aldev814** [LmeSzinc/AzurLaneAutoScript#3026] [Q&A] [answered]
What‘s the meaning of "Coolant" in the mail filter, I can't find the answer anywhere.

**Comment by bookbug666** [comment, marked as answer]
![image](https://github.com/LmeSzinc/AzurLaneAutoScript/assets/29662367/5afcd7dc-2c0d-4255-8c25-40fde23f71b6)

**Reply by aldev814** [reply]
Thanks!
